disclosure.md

```diff
@@ -0,0 +1,51 @@
+# Responsible Disclosure Policy
+
+18F Responsible Disclosure Policy 
+
+We take the security of our systems seriously, and we value the security community. The responsible disclosure of security vulnerabilities helps us ensure the security and privacy of our users.
+
+**Guidelines**  
+
+We require that all researchers:
+* Make every effort to avoid privacy violations, degradation of user experience, disruption to production systems, and destruction of data during security testing;
+* Perform research only within the scope set out below; 
+* Use the identified communication channels to report vulnerability information to us; and
+* Keep information about any vulnerabilities you’ve discovered confidential between yourself and 18F until we’ve had 90 days to resolve the issue.
+
+
+If you follow these guidelines when reporting an issue to us, we commit to:
+* Not pursue or support any legal action related to your research;
+* Work with you to understand and resolve the issue quickly (including an initial confirmation of your report within 72 hours of submission); 
+* Recognize your contribution on our Security Researcher Hall of Fame, if you are the first to report the issue and we make a code or configuration change based on the issue.
+
+
+**Scope**  
+* www.18F.gov
```

Comment:
There is no www.18f.gov website right now. 18f.gov and www.18f.gov redirect to 18f.gsa.gov. Right now, the endpoints at 18f.gov and www.18f.gov are (separate) cloud.gov apps that just redirect users. 18f.gsa.gov is a Federalist application, so is this putting Federalist into scope?